### PR TITLE
[Ubuntu] Modify node name with user

### PR DESCRIPTION
**Pull request by resouer**
`vcap` user is a test example user, it is confused, so I changed it to `vcap@xxx.xxx.xxx.xxx`

@dalanlan please review :)

### CODE REVIEW

**Comment by dalanlan** [comment]
This should apply to #L299 as well:)

**Comment by resouer** [comment]
@dalanlan Oh, yes I will fix the rest

**Comment by resouer** [comment]
Not sure why e2e is not satisfied to a doc fix .. sigh. Any tips about the 3 failures?

File: docs/getting-started-guides/ubuntu.md [hunk 1]
```diff
@@ -296,7 +296,7 @@ $ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -h
 Here are some examples:
 
 * upgrade master to version 1.0.5: `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -m 1.0.5`
-* upgrade node 10.10.103.223 to version 1.0.5 : `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -n 10.10.103.223 1.0.5`
+* upgrade node `vcap@10.10.103.223` to version 1.0.5 : `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -n vcap@10.10.103.223 1.0.5`
```
Comment (dalanlan) [inline docs/getting-started-guides/ubuntu.md]: Should be `$ KUBERNETES_PROVIDER=ubuntu ./kube-push.sh -n 10.10.103.223 1.0.5`

**Comment by dalanlan** [comment]
Moreover e2e is not required since it's a doc change.

**Comment by dalanlan** [comment]
lgtm

**Comment by resouer** [comment]
ping @smarterclayton , seems we can ship this now

**Comment by wojtek-t** [comment]
@k8s-bot unit test this please

**Pull request merged**